.eslintrc.js

```diff
@@ -73,12 +81,10 @@ module.exports = {
     react: {
       version: '18',
     },
-    'import/parsers': {
-      '@typescript-eslint/parser': ['.ts', '.tsx'],
-    },
```

Comment:
Assume this was removed for alias imports to work within mdx files? Its listed as part of configuration for [eslint-import-resolver-typescript](https://www.npmjs.com/package/eslint-import-resolver-typescript#configuration) but it looks to work fine without it. 👍

Reply:
Yeah it had some trouble reading files, and removing this fixed it. I made some changes since removing it, so we could potentially add it back if we want

Reply:
I added it back, and everything seems to work fine 👍🏻